app/resources/v1/__tests__/user.test.js

```diff
@@ -24,7 +28,7 @@ describe('PUT api/v1/users/:user_id', () => {
     app.put('/api/v1/users/:user_id', mockUserMiddleware, user.put)
   })
 
-  it('should respond with 204 user updates their own credentials', () => {
+  it.only('should respond with 204 user updates their own credentials', () => {
```

Comment:
We should remove this `.only` (since we only put it in temporarily to examine this one test)